tests/proxy_test.go

```diff
@@ -46,39 +40,49 @@ var _ = Describe("Kubevirt proxy", func() {
 	Context("/token endpoint", func() {
 
 		It("should fail if not authenticated", func() {
-			tokenUrl := fmt.Sprintf(tokenUrlTemplate, "test-vm")
+			tokenUrl := getTokenUrl("test-vm")
 			code, body, err := httpGet(tokenUrl, "", TestHttpClient)
 			Expect(err).ToNot(HaveOccurred())
-			Expect(code).To(Equal(http.StatusUnauthorized))
-			Expect(string(body)).To(ContainSubstring("authenticating token cannot be empty"))
+			Expect(code).To(Equal(http.StatusForbidden))
```

Comment:
Is this status required by K8S?

Reply:
What do you mean "Required"? The status is returned by k8s api server when a user does not have RBAC permission to access something.
Interestingly, if a request does not have any authentication token, k8s interprets the user as `system:anonymous`, ans still returns 403, instead of 404.

Reply:
Ok, my question was if this change has something to do with the move to being a K8S API extension. It looks like it.